_source/_posts/2022-04-28-gdpr-and-app-security.md

```diff
@@ -0,0 +1,148 @@
+---
+type: awareness
+github: https://github.com/oktadev/okta-dotnet6-webapi-example
+layout: blog_post
+title: "GDPR and application security"
+author: olabayo-balogun
+by: contractor
+communities: []
+description: "Learn more about GDPR rules and how they apply to your organization. Make sure your application contains the needed features to ensure high-level security."
+tags: [best-practices, security, privacy, gdpr]
+tweets:
+  - "GDPR rules can be long and esoteric, but it's important to know how to stay in compliance with the policy. Not only will you avoid fines for violations, but users will be more willing to trust your application. In other words: compliance is worth the work."
+  - "As the internet has grown, so too has the need to protect users from privacy invasions, fraud, and other types of abuse. If your organization or application is subject to GDPR rules, you'll need to know how to get and stay in compliance."
+image: blog/gdpr-and-app-security/gdpr-cover.jpg
+---
+
+As the internet has grown, so has the need to protect users from privacy invasions, fraud, or other types of abuse by attackers. The European Union's solution is the General Data Protection Regulation, or [GDPR](https://gdpr.eu/). First put into effect in 2018, the policy applies not only to the organizations in the EU but to any organization that uses or collects data from people in the EU.
+
+If your organization is subject to GDPR, you'll need to know how to stay in compliance. This article will break down the details of this policy so that you understand its potential effects on your application.
+
+## What to know about GDPR
+
+GDPR, which officially took effect on May 25, 2018, primarily encourages organizations to properly handle user data and privacy. Those not in compliance can face fines, in some cases [large ones](https://gdpr.eu/the-gdpr-meets-its-first-challenge-facebook/).
+
+### GDPR requirements
+
+GDPR compliance imposes multiple requirements on your organization. Here is what you must include in your application:
+
+- Enhanced application security.
```

Comment:
Can these be summarized? For example:

```
- Enhanced application security (e.g. HTTPS, Foo, Bar, etc).
```

Reply:
GDPR doesn't explicity define this, the closest would be a vague definition that requires end to end encryption and 2FA